src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -107,17 +106,15 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
   }
 
   when (RequestingOutputs) {
-    // Can't use the WorkflowOutputs type alias here since the @unchecked needs to be added to suppress
-    // compile time warnings.
-    case Event(outputs: Map[FullyQualifiedName@unchecked, CallOutput@unchecked], data) =>
+    case Event(WorkflowManagerWorkflowOutputsSuccess(id, outputs), data) =>
       // Outputs go to stdout
       outputOutputs(outputs)
       if (metadataOutputPath.isDefined) requestMetadata else issueReply
   }
   
   when (RequestingMetadata) {
-    case Event(response: WorkflowMetadataResponse, data) =>
-      val updatedData = outputMetadata(response) match {
+    case Event(r: WorkflowManagerWorkflowMetadataSuccess, data) =>
```

Comment:
While I'm all for it the switch to a terser variable name seems decidedly un-miguel-like